flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/RestHandlerSpecification.java

```diff
@@ -48,6 +49,6 @@ public interface RestHandlerSpecification {
      * @return Collection of supported API versions
      */
     default Collection<RestAPIVersion> getSupportedAPIVersions() {
-        return Collections.singleton(RestAPIVersion.V1);
+        return Collections.singleton(RuntimeRestAPIVersion.V1);
```

Comment:
Is this a common interface or specific to runtime?
- If common, it should not return a `RuntimeRestAPIVersion`
- If specific to runtime, the return type should be updated, as well as the name of the interface.